libc/pthread_cond_test_functions.h

```diff
@@ -34,7 +34,7 @@ typedef struct thread_args_t {
 } thread_args_t;
 
 
-thread_args_t thread_args;
+extern thread_args_t thread_args;
```

Comment:
@lukileczo I don't know how it worked without this, on host it won't even link due to multiple definitions of this symbol